test/currency_conversion_test.exs

```diff
@@ -90,4 +90,10 @@ defmodule CurrencyConversionTest do
              ]
     end
   end
+
+  describe "refresh_rates/0" do
+    test "refreshes rates" do
+      assert Converter.refresh_rates() == :ok
```

Comment:
We should test all conditions here:

* manually calling `refresh_rates` when interval is `manual`  works and does not `refresh` later
* manually calling `refresh_rates` when interval is for example 1 second does refresh later automatically
* manually calling `refresh_rates` when interval is `manual`  and loader fails works (returns error) and does not `refresh` later
* manually calling `refresh_rates` when interval is for example 1 second and loader fails works (returns error) does refresh later automatically